lib/index.js

```diff
@@ -280,7 +280,7 @@ Metalsmith.prototype.readFile = unyield(function*(file){
 
     if (frontmatter && utf8(buffer)) {
       try {
-        parsed = matter(buffer.toString());
+        parsed = matter(buffer.toString().replace(/\r\n/g, '\n'));
```

Comment:
Yeah, it's pretty clear Windows isn't _currently_ supported by gray-matter.

My point was that since it's gray-matter's lack of Windows support that's causing Metalsmith to fail here, it's probably best to fix that problem at the root instead of trying to work around it. Doing it that way is more in line with separation of concerns, and has the potential to benefit other projects besides Metalsmith which use gray-matter.